pkg/kubelet/cm/devicemanager/topology_hints_test.go

```diff
@@ -1006,6 +1006,7 @@ func getCommonTestCases() []topologyHintTestCase {
 				"testdevice": {
 					{ID: "Dev1"},
 					{ID: "Dev2"},
+					{ID: "Dev3", Topology: &pluginapi.TopologyInfo{Nodes: []*pluginapi.NUMANode{}}},
```

Comment:
This should also work for the following.
```
{ID: "Dev4", Topology: &pluginapi.TopologyInfo{Nodes: nil}},
```
Can you add this as a 4th test device, please?

Reply:
added.